template-view-duplicates.php

```diff
@@ -13,20 +13,23 @@
 $dt_duplicates = wp_unslash( $dt_contacts->get_all_duplicates() );
 ?>
 
-    <div id="content">
+    <div id="content" class="duplicates-page">
 
         <div id="inner-content" class="grid-x grid-margin-x">
 
             <main id="main" class="large-12 medium-12 cell" role="main">
                 <div class="bordered-box">
-                    <h3><?php esc_html_e( 'Duplicate Contacts', 'zume' ) ?></h3>
+                    <h3><?php esc_html_e( 'Duplicate Contacts', 'zume' ) ?><button class="help-button float-right" data-section="duplicates-template-help-text">
+                        <img class="help-icon" src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/help.svg' ) ?>"/>
+                    </button></h3>
 
+                    <?php //@// TODO: use an if statement to show a message if not duplicates were found. Else show the following table ?>
```

Comment:
Can you give this a try?

Reply:
No. Doing that is beyond me at present. Sorry.

Reply:
Ok. I don't disagree with you but please don't add TODOs for others to do in the code. 
We'll organize things to do in the issues.
So if you would please remove it here that would be great.

Reply:
Todo comment was removed :)